.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+name: Build main
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        java: [11, 8]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK ${{ matrix.java }}
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+
+      - name: Build Terraformgenerator
+        run: |
+          git config --global user.email "you@example.com"
+          git config --global user.name "Your Name"
+          chmod +x gradlew
```

Comment:
You can modify rights in git, so that this line would become pointless.
Additionally are the config options also not needed, unless the build task needs them?

Reply:
The task needs chmod to run.

Reply:
The task needs the build options, I'm not aware of a method to prevent this.

Reply:
Like I mentioned can the gradlew file get modified using a specific command in git, so that this manual chmod isn't nessesary.
Proof:
- https://github.com/botblock/JavaBotBlockAPI/blob/master/.github/workflows/release_actions.yml#L22
- https://github.com/botblock/JavaBotBlockAPI/blob/master/.github/workflows/release_actions.yml#L46

Reply:
On another node do I believe that this Action is not doing what you expect.
It uploads artifacts... But where? How can you use them? Is that even possible? What is the limit?

Reply:
This works perfectly fine I belive. About the output jars, I tested both the java 11 and the java 8 jar, worked fine without any issues. I don't think there's anything else needed to add into it.

Reply:
Yeah, a CI or Jenkins is so much better, but this does the job as well, if you are just looking to get the latest jar after a commit to test, and not compile it for yourself.